src/main/msp/msp_build_info.h

```diff
@@ -58,6 +58,7 @@
 #define BUILD_OPTION_ALTITUDE_HOLD              16422
 #define BUILD_OPTION_BATTERY_CONTINUE           16406
 #define BUILD_OPTION_CAMERA_CONTROL             16407
+#define BUILD_OPTION_CHIRP                      16426
```

Comment:
This needs to be added to cloud build API first. After - we can run the script to add it in this [read-only] file.